common/web/types/src/kmx/kmx-plus-builder/build-vars.ts

```diff
@@ -0,0 +1,61 @@
+import { constants } from "@keymanapp/ldml-keyboard-constants";
+import { KMXPlusData } from "../kmx-plus.js";
+import { build_strs_index, BUILDER_STRS } from "./build-strs.js";
+import { BUILDER_SECTION } from "./builder-section.js";
+
+
+interface BUILDER_VARS_ITEM {
+  type: number;
+  id: number; // str
+  value: number; // str
+  elem?: number; // elem, TODO-LDML
+};
+
+export interface BUILDER_VARS extends BUILDER_SECTION {
+  markers: number; // list, TODO-LDML
+  varCount: number;
+  varEntries: BUILDER_VARS_ITEM[];
+};
+
+/**
+* Builder for the 'vars' section
+*/
+export function build_vars(kmxplus: KMXPlusData, sect_strs: BUILDER_STRS /*, sect_elem: BUILDER_ELEM*/) : BUILDER_VARS {
+  if(!kmxplus.vars) {
+    return null;
+  }
+
+  const stringVars = kmxplus.vars.strings.map(v => <BUILDER_VARS_ITEM>{
```

Comment:
sneaky but nice way of avoiding `v => { return { ... } }`?

Reply:
a typed way anyway